src/Windowing/Silk.NET.Windowing.Common/Window.cs

```diff
@@ -189,10 +213,8 @@ public static void PrioritizeSdl()
         /// <param name="platform">The platform to add.</param>
         public static void Add(IWindowPlatform platform)
         {
-            if (!((List<IWindowPlatform>) Platforms).Contains(platform))
-            {
```

Comment:
Code style change isn't needed, recommended revert (generally I've operated under the assumption that code style is up to the decision of the codeowner, and I personally prefer braces for verbosity and explicitness)

Reply:
This is to make the `Add` call from within `DoLoadFirstPartyPlatformsViaReflection` not recursive.

Reply:
All you did is remove the brackets and use `_platforms` instead of casting?

Reply:
Yes

Reply:
Ok well can you add the brackets back.

Reply:
OH, sorry I didn't understand 😅 
of course.

Reply:
Thanks haha